xla/pjrt/gpu/se_gpu_pjrt_client_test.cc

```diff
@@ -161,9 +161,14 @@ static constexpr char const* kProgram = R"(HloModule HostTransfer
       ROOT result = f32[2] get-tuple-element(recv-done), index=0
     })";
 
+GpuClientOptions DefaultOptions() {
+  // Most test cases expect exactly 2 GPUs.
+  return GpuClientOptions{.allowed_devices = std::set<int>({0, 1})};
```

Comment:
We are not allowed to use C++20 yet, so no designated initializers.

Reply:
Random examples:

https://github.com/openxla/xla/blob/a403e7eb02788ea865e3780557b19cda9e2b5b05/xla/pjrt/tf_pjrt_client_test.cc#L43

https://github.com/openxla/xla/blob/38b38564b27b0146abf0f8131983874f1f5d8fe2/xla/service/gpu/autotuning/gemm_fusion_autotuner.cc#L307

Reply:
OK